tests/panels/test_sql.py

```diff
@@ -121,7 +121,13 @@ def test_param_conversion(self):
             .filter(group_count__lt=10)
             .filter(group_count__gt=1)
         )
-        list(User.objects.filter(date_joined=datetime.datetime(2017, 12, 22, 16, 7, 1)))
+        list(
+            User.objects.filter(
+                date_joined=datetime.datetime(
+                    2017, 12, 22, 16, 7, 1, tzinfo=datetime.timezone.utc
+                )
+            )
+        )
```

Comment:
change from naive datetime, which django will interpret with the default timezone, to aware